{
  "cna_info": {
    "cna": "Xiaomi",
    "total_cves": 8,
    "total_cves_scored": 8,
    "average_eas_score": 55.25,
    "overall_average_score": 55.25,
    "average_foundational_completeness": 23.25,
    "average_root_cause_analysis": 10,
    "average_software_identification": 0,
    "average_severity_context": 15,
    "average_actionable_intelligence": 3,
    "average_data_format_precision": 4,
    "rank": 108,
    "active_cna_count": 295,
    "percentile": 63.7
  },
  "recent_cves": [
    {
      "cveId": "CVE-2024-45351",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-26T13:17:07.014Z",
      "totalEasScore": 57,
      "scoreBreakdown": {
        "foundationalCompleteness": 25,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45352",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T02:02:34.511Z",
      "totalEasScore": 57,
      "scoreBreakdown": {
        "foundationalCompleteness": 25,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45354",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T06:25:56.581Z",
      "totalEasScore": 56,
      "scoreBreakdown": {
        "foundationalCompleteness": 24,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45355",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T06:48:07.672Z",
      "totalEasScore": 55,
      "scoreBreakdown": {
        "foundationalCompleteness": 23,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45356",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T07:11:41.754Z",
      "totalEasScore": 55,
      "scoreBreakdown": {
        "foundationalCompleteness": 23,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45353",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T06:12:57.059Z",
      "totalEasScore": 55,
      "scoreBreakdown": {
        "foundationalCompleteness": 23,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45347",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-06-23T09:34:38.676Z",
      "totalEasScore": 54,
      "scoreBreakdown": {
        "foundationalCompleteness": 22,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    },
    {
      "cveId": "CVE-2024-45361",
      "assigningCna": "Xiaomi",
      "datePublished": "2025-03-27T07:16:21.898Z",
      "totalEasScore": 53,
      "scoreBreakdown": {
        "foundationalCompleteness": 21,
        "rootCauseAnalysis": 10,
        "softwareIdentification": 0,
        "severityAndImpactContext": 15,
        "actionableIntelligence": 3,
        "dataFormatAndPrecision": 4
      }
    }
  ],
  "total_cves": 8
}